src/Tribe/Tickets.php

```diff
@@ -176,6 +176,113 @@ public function get_ticket( $event_id, $ticket_id ) {
 		}
 
 		/**
+		 * Returns the html for the delete ticket link
+		 *
+		 * @since TBD
+		 *
+		 * @param obj $ticket Ticket object
+		 */
+		public function get_ticket_delete_link( $ticket = null ) {
+			if ( empty( $ticket ) ) {
+				return;
+			}
+
+			if ( apply_filters( 'tribe_tickets_current_user_can_delete_ticket', true, $ticket->ID, $ticket->provider_class ) ) {
+				$delete_link = sprintf(
+					'<span><a href="#" attr-provider="%1$s" attr-ticket-id="%2$s" id="ticket_delete_%2$s" class="ticket_delete">'
+					. esc_html__( 'Delete Ticket', 'event-tickets' ) . '</a></span>',
```

Comment:
Element of personal preference here but I'd say - especially for simpletons like me - this line is easily mistaken for a subsequent `sprintf()` arg whereas it's actually part of the initial string ... there are a few ways we could make it clearer, including forming the string and passing it to `sprintf()` as a variable. Not a blocker.